packages/config/src/Autocomplete.ts

```diff
@@ -49,6 +49,11 @@ export interface Autocomplete extends BaseFeature<'Autocomplete'>, TemplateSetti
    */
   enableTrendingSearches: boolean
 
+  /**
+   * Shows/hides suggestions on mobile autocomplete
+   */
+  enableSuggestionsOnMobileAutocomplete: boolean,
```

Comment:
This option is already in config https://github.com/findify/findify-js/blob/6.17.0/packages/config/src/Autocomplete.ts#L20